src/test/java/com/github/dockerjava/netty/exec/UpdateSwarmCmdExecTest.java

```diff
@@ -0,0 +1,102 @@
+package com.github.dockerjava.netty.exec;
+
+import com.github.dockerjava.api.exception.DockerException;
+import com.github.dockerjava.api.exception.NotAcceptableException;
+import com.github.dockerjava.api.model.Swarm;
+import com.github.dockerjava.api.model.SwarmCAConfig;
+import com.github.dockerjava.api.model.SwarmDispatcherConfig;
+import com.github.dockerjava.api.model.SwarmOrchestration;
+import com.github.dockerjava.api.model.SwarmRaftConfig;
+import com.github.dockerjava.api.model.SwarmSpec;
+import com.github.dockerjava.api.model.TaskDefaults;
+import com.github.dockerjava.netty.AbstractNettySwarmDockerClientTest;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testng.ITestResult;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+import java.lang.reflect.Method;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.not;
+
+@Test(groups = "swarm-integration")
+public class UpdateSwarmCmdExecTest extends AbstractNettySwarmDockerClientTest {
+
+    public static final Logger LOG = LoggerFactory.getLogger(UpdateSwarmCmdExecTest.class);
+
+    @BeforeTest
+    public void beforeTest() throws Exception {
+        super.beforeTest();
+    }
+
+    @AfterTest
+    public void afterTest() {
+        super.afterTest();
+    }
+
+    @BeforeMethod
+    public void beforeMethod(Method method) {
+        super.beforeMethod(method);
+    }
+
+    @AfterMethod
+    public void afterMethod(ITestResult result) {
+        super.afterMethod(result);
+    }
+
+    public void updateSwarm() throws DockerException {
+        SwarmSpec firstSpec = new SwarmSpec().withName("firstSpec");
+
+        SwarmSpec secondSpec = new SwarmSpec()
+                .withName("secondSpec")
+                .withDispatcher(new SwarmDispatcherConfig()
+                        .withHeartbeatPeriod(10000000L)
+                ).withOrchestration(new SwarmOrchestration()
+                        .withTaskHistoryRententionLimit(100)
+                ).withCaConfig(new SwarmCAConfig()
+                        .withNodeCertExpiry(60 * 60 * 1000000000L /*ns */))
+                .withRaft(new SwarmRaftConfig()
```

Comment:
so would ask not to place `)` on new line as it seems simple fluent code that used everywhere with dot on new line.